Pilot/MessageSender.py

```diff
@@ -0,0 +1,266 @@
+""" Message sender module for the remote loggin system.
+    It provides the general interface of the message sender and
+    several implementations that e.g. allows to send the message
+    to a REST interface or to a MQ server using the Stomp protocol.
+"""
+
+import Queue
+import logging
+import stomp
+import requests
+
+
+class MessageSender(object):
+  """ General interface of message sender.
+  """
+
+  def sendMessage(self, msg, flag):
+    """ Must be implemented by children classes.
+    """
+    raise NotImplementedError
+
+
+def createMessageSender(senderType, params):
```

Comment:
There's a "pattern" in DIRAC code about the name of these types of functions, that is to use "factory", e.g. `messageSenderFactory`

Reply:
Ok. Im going to change it. Question of  taste but for me  a single function, XXXcreate() is more expressive than messageSenderFactory(...). or I would  prefer at least messageSenderFactory().create(....), but then you must create an additional class.

Reply:
changed